services/user-svc/main.go

```diff
@@ -24,8 +24,8 @@ func main() {
 
 		daprClient := common.MustNewDaprGRPCClient()
 
-		common.MustAddTopicEventHandler(server, user.UserRegisteredEventSubscription, user.HandleUserRegisteredEvent)
-		common.MustAddTopicEventHandler(server, user.UserUpdatedEventSubscription, user.HandleUserUpdatedEvent)
+		common.MustAddTopicEventHandler(server, user.RegisteredEventSubscription, user.HandleUserRegisteredEvent)
+		common.MustAddTopicEventHandler(server, user.UpdatedEventSubscription, user.HandleUserUpdatedEvent)
```

Comment:
```suggestion
		common.MustAddTopicEventHandler(server, user.RegisteredEventSubscription, user.HandleUserRegisteredEvent)
		common.MustAddTopicEventHandler(server, user.UpdatedEventSubscription, user.HandleUserUpdatedEvent)
```
Why was that changed? Not in scope of this pull request.

Reply:
Yes is not part of this pull request however it should be avoided that functions with the package name staten

Reply:
Yes is not part of this pull request however it should be avoided that functions start with the package name